Backend/app/dashboard/CommonAPIs/spec/RiderPlatform/Management/API/Ride.yaml

```diff
@@ -0,0 +1,304 @@
+imports:
+  Booking: Dashboard.Common
+  Customer: Dashboard.Common
+  Ride: Dashboard.Common
+  Summary: Dashboard.Common
+  Distance: Kernel.Types.Distance
+  HighPrecMeters : Kernel.Types.Distance
+  ShortId: Kernel.Types.Id
+  Money: Kernel.Types.Price
+  PriceAPIEntity: Kernel.Types.Price
+  HighPrecMoney: Kernel.Types.Price
+  Seconds: Kernel.Types.Time
+  UTCTime: Kernel.Prelude
+  FareProductType: Domain.Types
+  TripCategory: Domain.Types
+  VehicleVariant: Domain.Types.VehicleVariant
+  Centesimal: Kernel.Types.Centesimal
+  BaseUrl: Kernel.Prelude
+  GetRoutesResp: Kernel.External.Maps
+  MultipleRideSyncResp: Dashboard.Common.Ride
+  CancellationReasonCode: Dashboard.Common.Booking
+  Booking: Dashboard.Common
+  BookingStatus: Dashboard.RiderPlatform.Ride
+  Classification: Kernel.External.Ticket.Interface.Types
+
+module: Ride
+apis:
+  - GET:
+      endpoint: /list
+      auth: ApiAuth APP_BACKEND_MANAGEMENT RIDES RIDE_LIST
+      query:
+        - limit: Int
+        - offset: Int
+        - bookingStatus: BookingStatus
+        - rideShortId: (ShortId Ride)
+        - customerPhoneNo: Text
+        - driverPhoneNo: Text
+        - from: UTCTime
+        - to: UTCTime
+      response:
+        type: RideListRes
+  - GET:
+      endpoint: /rideinfo/{rideId}
+      auth: ApiAuth APP_BACKEND_MANAGEMENT CUSTOMERS RIDE_INFO_CUSTOMER
+      params:
+        rideId: Id Ride
+      response:
+        type: RideInfoRes
+  - GET:
+      endpoint: /{rideId}/info
+      auth:  ApiAuth APP_BACKEND_MANAGEMENT CUSTOMERS RIDE_INFO_CUSTOMER   # FIXME wrong auth need to be changed
+      params:
+        rideId: Id Ride
+      response:
+        type: ShareRideInfoRes
+  - GET:
+      endpoint: /{rideShortId}/rideInfo
```

Comment:
autogenerated names of these three apis are too confusing, can we rename?

`/{rideId}/info` to `GetShareRideInfo`
`/{rideShortId}/rideInfo` to `GetShareRideInfoByShortId` 
`/rideinfo/{rideId}` to `GetRideInfo`